main.go

```diff
@@ -215,6 +218,9 @@ var (
 
 func readContents(path string) ([]byte, error) {
 	if !strings.HasPrefix(path, "http://") && !strings.HasPrefix(path, "https://") {
+		if runtime.GOOS == "windows" {
+			path = filepath.Join(strings.Split(path, "/")...)
+		}
```

Comment:
This can be made more general (i.e., support other OSes that use non-'/'-filepath-separator, not just Windows) and simpler by using [`filepath.FromSlash`](https://godoc.org/path/filepath#FromSlash):

```Go
return readFile(filepath.FromSlash(path))
```